Formula/python35.rb

```diff
@@ -1,16 +1,8 @@
 class Python35 < Formula
   desc "Interpreted, interactive, object-oriented programming language"
   homepage "https://www.python.org/"
-  url "https://www.python.org/ftp/python/3.5.2/Python-3.5.2.tar.xz"
-  sha256 "0010f56100b9b74259ebcd5d4b295a32324b58b517403a10d1a2aa7cb22bca40"
-
-  # Patch for pyport.h macro issue
```

Comment:
why did you remove this? Is it fixed in 3.5.4?

Reply:
@drolando Yes, it's [fixed in 3.5.3][1].

[1]: https://docs.python.org/3/whatsnew/changelog.html#id126